website/src/components/validation/validation.test.ts

```diff
@@ -37,18 +37,41 @@ test("Running Validation Tests for VALIDATE_free_form", () => {
   ).toBe(false);
 });
 
+//This function is here to avoid hard coded years in the tests, by returning the current year plus a specified number of years as a string
+function thisYearPlus(years: number): string {
+  let year = new Date().getFullYear();
+  year += years;
+  return year.toString();
+}
+
 test("Running Validation Tests for VALIDATE_graduation", () => {
-  expect(VALIDATE_graduation("2021").validate).toBe(true);
-  expect(VALIDATE_graduation("2022").validate).toBe(true);
-  expect(VALIDATE_graduation("2023").validate).toBe(true);
-  expect(VALIDATE_graduation("2099").validate).toBe(true);
+  //Tests expected graduation years
+  expect(VALIDATE_graduation(thisYearPlus(0)).validate).toBe(true);
+  expect(VALIDATE_graduation(thisYearPlus(1)).validate).toBe(true);
+  expect(VALIDATE_graduation(thisYearPlus(2)).validate).toBe(true);
+  expect(VALIDATE_graduation(thisYearPlus(3)).validate).toBe(true);
+  expect(VALIDATE_graduation(thisYearPlus(4)).validate).toBe(true);
+  expect(VALIDATE_graduation(thisYearPlus(5)).validate).toBe(true);
+
+  //Tests graduation years slightly around the limit
+  expect(VALIDATE_graduation(thisYearPlus(-1)).validate).toBe(false);
+  expect(VALIDATE_graduation(thisYearPlus(6)).validate).toBe(false);
+  expect(VALIDATE_graduation(thisYearPlus(7)).validate).toBe(false);
+  expect(VALIDATE_graduation(thisYearPlus(10)).validate).toBe(false);
+  expect(VALIDATE_graduation(thisYearPlus(20)).validate).toBe(false);
 
+  //Tests bonkers years that should never be valid
   expect(VALIDATE_graduation("").validate).toBe(false);
```

Comment:
```suggestion
  expect(VALIDATE_graduation("").validate).toBe(false);
  expect(VALIDATE_graduation(NaN).validate).toBe(false);
  expect(VALIDATE_graduation(2021.24).validate).toBe(false); //floating point
```

more possible edge cases